datahost-graphql/src/tpximpact/catql.clj

```diff
@@ -340,6 +345,33 @@
 (defmethod ig/init-key ::schema [_ opts]
   (load-schema opts))
 
+(defn- facet-label-query [pred]
+  {:prefixes default-prefixes
+   :select-distinct '[?facet ?facet_label]
+   :where ['[?catalog_uri (cat :dcat/record :foaf/primaryTopic) ?id]
+           [:optional
+            [['?id pred '?facet]
+             '[?facet :rdfs/label ?facet_label]]]]})
+
+(defn- load-facet-labels! [repo]
+  (let [facet-preds [:dcat/theme :dcterms/publisher :dcterms/creator]
+        naive-pluralize (comp keyword #(str % "s") name)]
+    (->> (for [pred facet-preds]
+           [(naive-pluralize pred) (->> (query repo (facet-label-query pred))
+                                        (remove empty?)
+                                        (map (juxt :facet :facet_label))
+                                        (into {}))])
+         (into {}))))
+
+(defn- initialisation-side-effects! [_sys]
+  (when-not @facet-labels-store
+    (reset! facet-labels-store
+            ;; TODO: endpoint repo should probably be injected from conf
+            (try (load-facet-labels! (repo/sparql-repo "https://beta.gss-data.org.uk/sparql"))
+                 (catch Exception ex
+                   (log/error "Could not load facet labels" ex)
+                   {})))))
```

Comment:
I don't think we want to swallow the exception here, we should probably try and crash the process:

`(throw (ex-info "Could not load facet labels" {} ex))`

Otherwise we'll think everything is hunky dory and at some unspecified point in the future get a bug report to debug about "why aren't the labels rendering"?

If we crash the process we can for now kick the can over to the process being restarted.

This isn't of course the final solution, but will be good enough for now.  

A final solution will involve using the graph modified times and/or the endpoint version tags in drafter to test if the labels graph has changed since last time; and reloading the labels lazily if they have been changed.